ganymede/models/sources.yml

```
@@ -10,4 +10,12 @@ sources:
       - name: retailers
       - name: stores
       - name: statcan_food_prices
+  - name: static_resources
+    schema: public
+    tables:
+      - name: censusdivisions
```

```suggestion
      - name: census_divisions
```

Let's lower snake case everything for consistency.